Templates/ServiceBusQueueTrigger-NodeJS/index.js

```diff
@@ -0,0 +1,4 @@
+﻿module.exports = function(context, myQueueItem) {
+    context.log('Node.js ServiceBus queue trigger function processed message', myQueueItem);
+    context.done(null, OutPutQueueItem);
```

Comment:
Good catch - I'm testing it in prod and already caught that. I'll make sure it runs e2e.